templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  
+  <form action="{% url 'login' %}" method="POST">
+    {{ form.as_p }}
+    
+    {% csrf_token %}
+    <input type="submit" value="Submit">
+    <input type="hidden" name="next" value="{{ next }}">
```

Comment:
The 'next' input field is used to redirect users after a successful login. Ensure that the 'next' context variable is correctly passed to the template when rendering the login page.